consensus.md

```diff
@@ -52,74 +52,69 @@ Note that the value of L1_Proof_Height is less than L1_Block_Height.
 Example: _R_15[Alice, 100, 102, 20]_ means the generation is 15, the aggregator is _Alice_, the generation of the L1 bock used as proof is 100, the generation of the L1 bock that included the rollup is 102, and the nonce equals 20.
 
 ## The Canonical Chain
-The POBI protocol allows any aggregator to publish rollups to the management contract. This means that short-lived forks are a normal part of the protocol. The ObscuroVM running inside the TEE of every node must be able to deterministically select one of the forks as the canonical chain and only append a rollup on top of that. This means that the TEE must receive all the relevant content of the L1 blocks, and the logic must be identical on all nodes.
+The POBI protocol allows any aggregator to publish rollups to the management contract, so short-lived forks are a normal part of the protocol. However, the ObscuroVM running inside the TEE of every node must be able to deterministically select one of the forks as the canonical chain and only append a rollup on top of that. This means that the TEE must receive all the relevant content of the L1 blocks, and the logic must be identical on all nodes.
```

Comment:
I don't think this is a "however" statement

Reply:
The POBI protocol allows any aggregator to publish rollups to the management contract, so short-lived forks are a normal part of the protocol. The forks cannot be long-living during normal functioning because the ObscuroVM running inside the TEE of every node will deterministically select one of the forks as the canonical chain and only append a rollup on top of that.  Because the logic is identical on all nodes and the TEEs receive all the relevant content of the L1 blocks, there cannot be any competing forks more than one rollup deep unless there is a hack.